ivy_tests/test_ivy/test_frontends/test_paddle/test_random.py

```diff
@@ -7,6 +7,64 @@
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 
 
+# --- Helpers --- #
+# --------------- #
+
+
+@st.composite
+def _multinomial_helper(draw):
+    input_dtype_and_x = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("valid"),
+            shape=helpers.get_shape(min_num_dims=1, max_num_dims=2, min_dim_size=2),
+        )
+    )
+    num_samples = draw(st.integers(min_value=1, max_value=10))
+    if num_samples > 2:
+        replacement = True
+    else:
+        replacement = draw(st.booleans())
```

Comment:
Can you explain why the replacement has to be True for values bigger than 2 ?

Reply:
The number of dimensions of the input probability distribution (`x` array in this case) should be <= 2, this is the max all of the frameworks allow. Which is the reason I set `max_num_dims=2`. So if we have 2 dims we can't take a larger sample than population when `replacement=False`, hence whenever `num_samples > 2` we will set `replacement=True`.